.circleci/config.yml

```diff
@@ -255,15 +258,19 @@ workflows:
       - build-memif-test:
           requires:
             - sanity-check
-      - build:
-          requires:
-            - sanity-check
       - packet-deploy:
           requires:
             - sanity-check
       - packet-integration-tests:
           requires:
-            - build
+            - build-nsmd
```

Comment:
Do we really require all of these?   Looking below it looks like:

build-nsmd
build-nsmdp
build-nsmd-k8s
build-nsc
build-icmp-responder-nse